pkg/sql/sqlbase/structured.proto

```diff
@@ -536,6 +536,11 @@ message ComputedColumnSwap {
   option (gogoproto.equal) = true;
   optional uint32 new_column_id = 1 [(gogoproto.nullable) = false, (gogoproto.casttype) = "ColumnID"];
   optional uint32 old_column_id = 2 [(gogoproto.nullable) = false, (gogoproto.casttype) = "ColumnID"];
+  // inverse_expr represents the computed expression for the old column to
```

Comment:
This comment doesn't make that much sense. Something like 

"InverseExpr is the expression to compute values for the old column once it is swapped for the new column" is simpler.